lib/shortcode/jetpack-portfolio.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * Creates the view for Jetpack's contact form
+ */
+
+add_action( 'init', 'sandwich_jetpack_portfolio', 11 );
+
+function sandwich_jetpack_portfolio() {
+
+	// Check if Shortcake exists
+	if ( ! function_exists( 'shortcode_ui_register_for_shortcode') ) {
+		return;
+	}
+	
+	shortcode_ui_register_for_shortcode(
+        'portfolio',
+        array(
+            'label' => __( 'Jetpack' , 'jetpack' ) . ' ' . _x( 'Portfolio', 'Module Name', 'jetpack' ),
+            'listItemImage' => 'dashicons-portfolio',
+            'attrs' => array(
+                array(
+                    'label' => 'Display Portfolio Types',
```

Comment:
Turn all translatable strings to: `__( 'Display Portfolio Types', 'pbsandwich' )`